Gopkg.toml

```diff
@@ -0,0 +1,41 @@
+# Gopkg.toml example
+#
+# Refer to https://golang.github.io/dep/docs/Gopkg.toml.html
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
+
+
+
+[[constraint]]
+  name = "k8s.io/client-go"
+  version = "kubernetes-1.12.3"
+
+[[constraint]]
+  name = "github.com/giantswarm/apiextensions"
+  branch = "kubernetes-1.12.3"
```

Comment:
Is this correct? I don't see branch nor tag with `kubernetes-1.12.3` in `apiextensions`. There is `kubernetes-1.13.1`, but what about above `client-go` then?

Reply:
@tuommaki we as team batman take over the issue from team-magic, @kopiczko delete his branch from last week. https://github.com/giantswarm/apiextensions/pull/167
So, yes. I need to replace it with kubernetes-1.13.1.

I think we need to change `client-go` constraint as same as apiextension, doesn't it?

Reply:
> I think we need to change client-go constraint as same as apiextension, doesn't it?

Yeah, I think so.